testdata/jsight_0.3/ser-to-openapi/req.jschema.rules.nullable_03/nullable.false.enum.openapi.todo

```diff
@@ -0,0 +1,113 @@
+{
+  "openapi": "3.0.3",
+  "info": {
+    "title": "",
+    "version": ""
+  },
+
+  "paths": {
+    "/request/jschema/rules/nullable/enum": {
+      "get": {
+        "responses": {
+
+          "200": {
+            "description": "",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "example": "white",
+                  "enum": ["white", "blue", "red"]
+                }
+              }
+            }
+          },
+
+          "201": {
+            "description": "",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "example": -2.1,
+                  "enum": [-3, -2.1, 1.2, true, false, "-3", "0", "1.2", "string", "true"]
+                }
+              }
+            }
+          },
+
+          "202": {
+            "description": "",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "example": null,
+                  "enum": ["white", "blue", "red", null]
+                }
+              }
+            }
+          },
+
+          "203": {
+            "description": "",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "type": "object",
+                  "properties": {
+                    "sEnum1": {
+                      "example": "white",
+                      "enum": ["white", "blue", "red"],
+                      "nullable": true
+                    },
+                    "sEnum2": {
+                      "example": -2.1,
+                      "enum": [-3, -2.1, 1.2, true, false, "-3", "0", "1.2", "string", "true"],
+                      "nullable": true
+                    },
+                    "sEnum3": {
+                      "example": null,
+                      "enum": ["white", "blue", "red", null],
+                      "nullable": true
+                    }
+                  },
+                  "required": ["sEnum1", "sEnum2", "sEnum3"],
+                  "additionalProperties": false
+                }
+              }
+            }
+          },
+
+          "204": {
+            "description": "",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "type": "array",
+                  "items": {
+                    "anyOf": [
+                      {
+                        "example": "white",
+                        "enum": ["white", "blue", "red"],
+                        "nullable": true
+                      },
+                      {
+                        "example": -2.1,
+                        "enum": [-3, -2.1, 1.2, true, false, "-3", "0", "1.2", "string", "true"],
+                        "nullable": true
```

Comment:
nullable is false here

Reply:
https://github.com/jsightapi/jsight-api-core/pull/15/commits/c7c48a17ea72184bb0adb9ebbe483908e1e14df5